service/src/main/java/bio/terra/cbas/runsets/types/CbasBoolean.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.cbas.runsets.types;
+
+public class CbasBoolean implements CbasValue {
+  private final Boolean value;
+
+  /*
+  Private constructor. Use 'parse' instead.
+   */
+  private CbasBoolean(Boolean value) {
+    this.value = value;
+  }
+
+  @Override
+  public Object asCromwellInput() {
+    return value;
+  }
+
+  public static CbasBoolean parse(Object value) throws CoercionException {
+    if (value instanceof Boolean b) {
+      return new CbasBoolean(b);
+    } else {
+      throw new TypeCoercionException(value, "Int");
+    }
+  }
+}
```

Comment:
[Not a Blocker] Why do we need different files for all of these different input types?

Reply:
You can thank Java... it needs every class to be in its own appropriately-named file. Or are you asking why we want separate java classes for every type?